cmd/buildah/images.go

```diff
@@ -24,22 +46,35 @@ var (
 			Usage: "do not print column headings",
 		},
 		cli.BoolFlag{
-			Name:  "notruncate",
+			Name:  "no-trunc, notruncate",
 			Usage: "do not truncate output",
 		},
 		cli.BoolFlag{
 			Name:  "json",
 			Usage: "output in JSON format",
 		},
+		cli.BoolFlag{
+			Name:  "digests",
+			Usage: "show digests",
+		},
+		cli.StringFlag{
+			Name:  "format",
+			Usage: "pretty-print images using a Go template. will override --quiet",
+		},
+		cli.StringFlag{
+			Name:  "filter, f",
+			Usage: "filter output based on conditions provided (default [])",
+		},
 	}
-	imagesDescription = "Lists locally stored images."
+
+	imagesDescription = "lists locally stored images."
 	imagesCommand     = cli.Command{
 		Name:        "images",
-		Usage:       "List images in local storage",
+		Usage:       "list images in local storage",
 		Description: imagesDescription,
 		Flags:       imagesFlags,
 		Action:      imagesCmd,
-		ArgsUsage:   " ",
+		ArgsUsage:   "",
```

Comment:
This causes the help output to include `[arguments...]` after `[command options]` in the usage section.  It's preferable to indicate what type of value we expect to be given, as we do for the other commands.